Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs

```diff
@@ -285,18 +285,18 @@ private void AddSmiteVerbs(GetVerbsEvent<Verb> args)
             {
                 Text = "admin-smite-remove-hands-name",
                 Category = VerbCategory.Smite,
-                Icon = new SpriteSpecifier.Rsi(new ("/Textures/Fluids/vomit_toxin.rsi"), "vomit_toxin-1"),
+                Icon = new SpriteSpecifier.Rsi(new("/Textures/Fluids/vomit_toxin.rsi"), "vomit_toxin-1"),
                 Act = () =>
                 {
                     _vomitSystem.Vomit(args.Target, -1000, -1000); // You feel hollow!
-                    var organs = _bodySystem.GetBodyOrganComponents<TransformComponent>(args.Target, body);
```

Comment:
this should just be changed to return `Entity<T>`

Reply:
oh, can I do that?

Reply:
ah, I can't just do that... I'm assuming you want me to make a version of GetBodyOrganComponents that just returns a List of Entity<T> instead

Reply:
Is uh, whatever I did, what you wanted?

Reply:
add OrganComponent to the Entity, it supports multiple components

Reply:
![image](https://github.com/user-attachments/assets/fc15d5cf-1a0f-4029-9a7b-ccb4374dede8)
hmmmm